modules/mono/editor/bindings_generator.cpp

```diff
@@ -3973,4 +3973,25 @@ void BindingsGenerator::handle_cmdline_args(const List<String> &p_cmdline_args)
 	}
 }
 
+bool BindingsGenerator::is_generating_glue(const List<String> &p_cmdline_args) {
```

Comment:
Can this be deduplicated with the loop in the method above, which checks basically the same thing?

Reply:
I can set some internal class state with the needed data in this loop so it can be used later on, yes. This isn't a particularly big loop so the performance impact of doing it twice is minimal, but being able to parse command line arguments up front and then later on use that information is probably a useful thing to have, now that I think about it.

Reply:
I've split this off into its own PR at #76659, and refactored the code in the process to not duplicate code.